frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -536,177 +504,115 @@ export function Experiment(): JSX.Element {
-                                    <span>
-                                        ~{experimentData.parameters?.recommended_sample_size}{' '}
-                                        <span className="text-muted">persons</span>
-                                    </span>
-                                </Col>
+                    <Row>
+                        {showWarning &&
+                            experimentResults &&
+                            ((experimentInsightType === InsightType.TRENDS && areCountResultsSignificant) ||
+                                (experimentInsightType === InsightType.FUNNELS && areConversionResultsSignificant)) && (
+                                <Row align="middle" className="significant-results">
+                                    <Col span={19} style={{ color: '#497342' }}>
+                                        Experiment results are significant. You can end your experiment now or let it
+                                        run until completion.
+                                    </Col>
+                                    <Col span={5}>
+                                        <Button style={{ color: '#497342' }} onClick={() => setShowWarning(false)}>
+                                            Dismiss
+                                        </Button>
+                                    </Col>
+                                </Row>
                             )}
-                            <Col>
-                                <div className="card-secondary mt">Variants</div>
-                                <ul className="variants-list">
-                                    {experimentData.parameters?.feature_flag_variants?.map(
-                                        (variant: MultivariateFlagVariant, idx: number) => (
-                                            <li key={idx}>{variant.key}</li>
-                                        )
+                    </Row>
+                    <Row>
+                        <Collapse className="full-width" defaultActiveKey="experiment-details">
+                            <Collapse.Panel header={<b>Experiment details</b>} key="experiment-details">
+                                <ExperimentPreview
+                                    experiment={experimentData}
+                                    trendCount={trendCount}
+                                    exposure={exposure}
+                                    sampleSize={sampleSize}
+                                    runningTime={runningTime}
+                                    conversionRate={conversionRate}
+                                />
+                                <Col span={8} className="mt ml">
+                                    <div className="mb-05">
+                                        <b>Experiment progress</b>: {experimentProgressPercent.toFixed(1)}%
+                                    </div>
+                                    <Progress
```

Comment:
should be loading / invisible in the beginning? (and for drafts resolve to 0?) 

(my latest change kinda goes against the drafts resolve to 0 thing.)

Reply:
I just removed it for draft experiments, didn't make sense to have it there